include/asterisk/frame.h

```diff
@@ -335,6 +335,7 @@ enum ast_control_frame_type {
 	AST_CONTROL_STREAM_RESTART = 1002,	/*!< Indicate to a channel in playback to restart the stream */
 	AST_CONTROL_STREAM_REVERSE = 1003,	/*!< Indicate to a channel in playback to rewind */
 	AST_CONTROL_STREAM_FORWARD = 1004,	/*!< Indicate to a channel in playback to fast forward */
+	AST_CONTROL_PLAYBACK_BEGIN = 1005,	/*!< Indicate to a dialing interface that playback of an audio file should begin on the dialing channel. Currently only supported by app_dial. */
```

Comment:
No, the frames are not for a channel in playback, they're for `app_dial` which is dialing channel(s). It's not really like any type of frame that exists currently. I don't know that I thought about the wording that much but I don't think it's inaccurate. If you can suggest a better phrasing I'm happy to update it.

Reply:
Well, if this is not for a channel in playback then the value (and any associated comments) should be split out into their own section. I would also suggest the enum value being in it's own range and not pick then next AST_CONTROL_STREAM_xxx value.  What I see is AST_CONTROL_STREAM_xxx are the 1000's, the AST_CONTROL_RECORD_xxx values are the 1100's.  Maybe set the AST_CONTROL_PLAYBACK_xxx's @ 1150's or 1200's

Reply:
Well, it is somewhat related to "manipulating streams on channels", only it's starting a stream rather than manipulating an existing one as the other ones do.

@jcolp commented in November "There is no existing control frame specifically for this. There are other playback stream control frames, provided that it is done in a generic fashion and documented a control frame could be added alongside these to start playback.", which is why I added it where I did. Maybe someone from the Asterisk team can chime in if it should be in a new category or with the other stream control frames?

Reply:
AST_CONTROL_STREAM_* is probably misnamed realistically, as it's controlling playback on the channel and not really what we know of as media streams today. I don't have a problem with it being with the other stream control frames, they're all related to playback fundamentally.